src/core/Entry.cpp

```diff
@@ -325,25 +325,28 @@ QString Entry::totp() const{
 
         int rc = oath_init();
```

Comment:
It doesn't matter what Q_UNUSED does. Don't try to be smarter than the qt developers if you use their api.
https://doc.qt.io/qt-5/qtglobal.html#Q_UNUSED